CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 2.32.5.20250212
+This release includes:
+* Fluent Bit [1.9.10](https://github.com/fluent/fluent-bit/tree/v1.9.10)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.4
+* Amazon Kinesis Streams for Fluent Bit 1.10.1
```

Comment:
1.10.2?

Reply:
based on what I see in the previous changelog block below... 

```
* Amazon CloudWatch Logs for Fluent Bit 1.9.4
* Amazon Kinesis Streams for Fluent Bit 1.10.2
* Amazon Kinesis Firehose for Fluent Bit 1.7.2
```

Reply:
we don't appear to have updated linux.version, the versions in this changelog entry should be accurate for what we're actually building with